sdk/nodejs/package.json

```diff
@@ -31,5 +31,8 @@
     },
     "pulumi": {
         "comment": "Do not remove. Marks this as as a deployment-time-only package"
+    },
+    "resolutions": {
+        "grpc/node-pre-gyp": "^0.11.0"
```

Comment:
This uses's yarns `resolutions` feature to override the dependency that grpc has on node-pre-gyp to the newer version which has this fix: https://github.com/mapbox/node-pre-gyp/pull/408.  Without this, node-pre-gyp tries to build twice when the download fails on node 11, because of a breaking change that they made.

Reply:
We can remove this once prebuilt binaries are present for grpc or they move to use the newer version of `node-pre-gyp`.